lms/static/js/views/image_field.js

```diff
@@ -0,0 +1,230 @@
+;(function (define) {
+    'use strict';
+    define([
+        'gettext', 'jquery', 'underscore', 'backbone', 'js/views/fields',
+        'text!templates/fields/field_image.underscore',
+        'backbone-super', 'jquery.fileupload'
+    ], function (gettext, $, _, Backbone, FieldViews, field_image_template) {
```

Comment:
None of this code was changed, correct? It's simply a move of code out of fields.js.